README.md

```diff
@@ -1 +1,158 @@
-# OOP-school-library
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>OOP School Library</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👤 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 OOP School Library <a name="about-project"></a>
+
+**OOP School Library** is a backend app that allows you keeping track of all the records of books in a library and who borrows them.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Add new students or teachers.**
+- **Add new books.**
+- **Save records of who borrowed a given book and when.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- An IDE e.g [Visual studio code](https://code.visualstudio.com/).
+- [A terminal](https://code.visualstudio.com/docs/terminal/basics).
+
+### Setup
+### Setup
+
+Clone this repository to your desired folder or download the Zip folder:
+
+```
+git clone https://github.com/berylcl/OOP-school-library.git
+```
+
+- Navigate to the location of the folder in your machine:
+
+**``you@your-Pc-name:~$ cd OOP-School-Library``**
+
+### Install
+
+To install all dependencies, run:
+
+```
+bundle install
+```
+
+### Usage
+
+To run the project, follow these instructions:
+
+- After Cloning this repo to your local machine.
+- To get it running in your terminal, run it in your IDE
+
+### Run tests
+
+To run tests, run the following command:
+
+```
+rubocop
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👤 Author <a name="author"></a>
+
+## 👤 Author <a name="author"></a>
```

Comment:
- [ ] You have the`Author` section repeated in your readme file so make sure you remove one to make your readme file professional :blush: